src/ophyd_async/epics/adandor/_andor_io.py

```diff
@@ -16,23 +16,30 @@ class Andor2TriggerMode(StrictEnum):
     SOFTWARE = "Software"
 
 
-class ImageMode(StrictEnum):
+class Andor2ImageMode(StrictEnum):
```

Comment:
Could we make this one subclass the ADBase one and add `FAST_KINETICS` to it?

Reply:
I just tried this and this is not allowed:

> subclassing an enumeration is allowed only if the enumeration does not define any members.

Source: https://docs.python.org/3/howto/enum.html#restricted-enum-subclassing

In this case, I'd probably just leave this as-is, until we have `SupersetEnum` available.

Reply:
I think `SupersetEnum` is the wrong way for this case, we need to make the baseclass do `SubsetEnum` to say that subclasses are allowed to add their own extra image modes. However if we never use the baseclass imagemode anywhere then what is here at the moment is fine.